src/components/AnnotationWorkerMenu.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card v-if="tool" class="menu">
+  <v-card :class="{ menu: true, loaded: !fetchingWorkerInterface }" v-if="tool">
```

Comment:
I think you can go with 
```
<v-card
  class="menu",
  :class="{ loaded: !fetchingWorkerInterface}"
>
```
as menu is static and always true.

It should be rendered as:
```
<v-card class="menu loaded"> // if fetchingWorkerInterface = false
<v-card class="menu"> // if fetchingWorkerInterface = true
```